lib/Dialect/ESI/cosim/cosim_dpi_server/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+find_package(CapnProto CONFIG PATHS "${CMAKE_SOURCE_DIR}/ext")
```

Comment:
should this be inside the `if`? it seems like there's no need to look for CapnProto if the `ESI_COSIM` flag isn't set